### DISCUSSION TITLE
Support for adding TipTap extensions to the Flux editor component

### DISCUSSION THREAD

**Discussion by RezaODB** [livewire/flux#873] [Feature Requests]
Would be nice to have tables in the editor...

**Comment by jeffchown** [comment]
@RezaODB There is a TipTap extension for tables: https://tiptap.dev/docs/editor/extensions/nodes/table

To my knowledge though, it is not yet possible to add TipTap extensions to the Flux editor component.
(per https://github.com/livewire/flux/issues/746)

**Comment by RezaODB** [comment]
"To my knowledge though, it is not yet possible to add TipTap extensions to the Flux editor component." 
Well... I guess the topic is now: Support for adding TipTap extensions to the Flux editor component.

**Reply by jeffchown** [reply]
This has also been discussed here https://github.com/livewire/flux/discussions/739

**Comment by joshhanley** [comment]
@RezaODB thanks for the feature request! As discussion #739 is similar, I've renamed it to be `Rich text editor: add support for plugins`. Can you add your support there? I'm going to close this one in favour of that one.
